### PR TITLE
URBANIA TESTING!

### PR DESCRIPTION
**Pull request by Timmeye**
Sample text, multiplayer testing for the new urbania map (hope we have a test in under 3 years)

THIS IS WIP NOT ACTUALLY DONE YET!!!

### CODE REVIEW

File: config/maps.txt [hunk 1]
```diff
@@ -13,7 +13,10 @@ Format:
 endmap
 
 map lv624
-	default
+		default
```
Comment (DominikPanic) [inline config/maps.txt]: Fix the whitespace here

**Comment by Timmeye** [comment]
The map will be done REALLY soon like under 10 days probably

File: config/maps.txt (hunk 1 above)
Comment (spookydonut) [inline config/maps.txt]:
```suggestion
	default
```

File: _maps/urbania.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "map_name": "urbania",
+    "map_path": "map_files/urbania",
+    "map_file": "urbania.dmm",
+    "shuttles": {
+    },
+    "announce_text": "A urban settlement a sent out a distress beacon, the planet seems to be earthen with trees and grass growing"
```
Comment (spookydonut) [inline _maps/urbania.json]:
```suggestion
    "announce_text": "A urban settlement has sent out a distress beacon, the planet seems to be earthen with trees and grass growing"
```

**Comment by Rohesie** [comment]
What's the current status of the map?
Does it have two LZs and does it support Crash?

**Comment by Timmeye** [comment]
yeah just need to add more buildings then polish

**Comment by Alterist** [comment]
you should just  cut the right half of the map, center it and surround it with cave walls. There's no point in having empty fields like that when you can make a more compact, more interesting map

**Comment by Timmeye** [comment]
Yeah I will cut the right half off, I wont wanna build a lot more buildings

**Comment by sunofang** [comment]
This map lacks flavour, Else its quite good.

File: _maps/urbania.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "map_name": "urbania",
+    "map_path": "map_files/urbania",
+    "map_file": "urbania.dmm",
+    "shuttles": {
```
Comment (spookydonut) [inline _maps/urbania.json]:
```suggestion
```

File: _maps/urbania.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+    "map_name": "urbania",
+    "map_path": "map_files/urbania",
+    "map_file": "urbania.dmm",
+    "shuttles": {
+    },
```
Comment (spookydonut) [inline _maps/urbania.json]:
```suggestion
```

**Comment by spookydonut** [comment]
@Timmeye merge conflict

**Comment by Timmeye** [comment]
changed the config file maybe its fixed?

**Comment by Rohesie** [comment]
It's not conflicting anymore, but it needs more work.

**Comment by spookydonut** [comment]
Is this ready for testing again? @Timmeye

**Comment by spookydonut** [comment]
stale closing

**Pull request closed without merging**